tests/structured_conf/test_structured_config.py

```diff
@@ -1103,164 +1103,125 @@ def test_container_as_element_type_invalid(
             cfg[node] = value
 
     @pytest.mark.parametrize(  # type: ignore
-        "obj,cfg2,expected",
+        "obj,obj2,expected",
         [
             (
                 "ContainerInList",
-                OmegaConf.create(
-                    {"list_with_list": [[4, 5]], "list_with_dict": [{"foo": 4}]}
-                ),
-                {
-                    "list_with_list": [[4, 5]],
-                    "list_with_dict": [{"foo": 4}],
-                },
+                {"list_with_list": [[4, 5]], "list_with_dict": [{"foo": 4}]},
+                {"list_with_list": [[4, 5]], "list_with_dict": [{"foo": 4}]},
             ),
             (
                 "ContainerInDict",
-                OmegaConf.create(
-                    {
-                        "dict_with_list": {"foo": [3, 4]},
-                        "dict_with_dict": {"foo": {"var": 4}},
-                    }
-                ),
+                {
+                    "dict_with_list": {"foo": [3, 4]},
+                    "dict_with_dict": {"foo": {"var": 4}},
+                },
                 {
                     "dict_with_list": {"foo": [3, 4]},
                     "dict_with_dict": {"foo": {"var": 4}},
                 },
             ),
             (
                 "ComplexList",
-                OmegaConf.create(
-                    {
-                        "list": [[{"foo": 13}]],
-                    }
-                ),
-                {
-                    "list": [[{"foo": 13}]],
-                },
+                {"list": [[{"foo": 13}]]},
+                {"list": [[{"foo": 13}]]},
             ),
             (
                 "ComplexDict",
-                OmegaConf.create(
-                    {
-                        "dict": {"foo2": {"var2": [3, 4]}},
-                    }
-                ),
-                {
-                    "dict": {"foo": {"var": [1, 2]}, "foo2": {"var2": [3, 4]}},
-                },
+                {"dict": {"foo2": {"var2": [3, 4]}}},
+                {"dict": {"foo": {"var": [1, 2]}, "foo2": {"var2": [3, 4]}}},
             ),
         ],
     )
     def test_container_as_element_type_merge(
-        self, class_type: str, obj: str, cfg2: Any, expected: Any
+        self, class_type: str, obj: str, obj2: Any, expected: Any
     ) -> None:
         module: Any = import_module(class_type)
         class_ = getattr(module, obj)
         cfg = OmegaConf.merge(class_)
+        cfg2 = OmegaConf.create(obj2)
         res = OmegaConf.merge(cfg, cfg2)
```

Comment:
In fact, you can merge the dict directly so you don't even need to create here :)

Reply:
hmm, I didn't notice it was transformed into dictconfig when merging, good to know.